liberty-managed/src/main/java/io/openliberty/arquillian/managed/WLPManagedContainer.java

```diff
@@ -295,7 +308,64 @@ public void run() {
       }
    }
 
-	private List<String> parseJvmArgs(String javaVmArguments) {
+//    /***
+//     * Searches and parses existing server.env files to the ProcessBuilder.environment prior to server start-up
+//     *
+//     * The server management script searches for server.env files in two locations:
+//     * ${wlp.install.dir}/etc/server.env and ${server.config.dir}/server.env. If both files are present, the
+//     * contents of the two files are merged; values in the server-level file take precedence over values in the
+//     * runtime-level file.
+//     *
+//     * The allowed server.env variables shall be alphanumeric with possible underscores
+//     * {@link [com.ibm.ws.kernel.boot.ws-server.]server.bat[#readServerEnv()]}
+//     *
+//     * @param environment
+//     * @throws LifecycleException
+//     * @throws IOException
+//     */
+//    private void parseServerEnv(Map<String, String> environment) throws LifecycleException, IOException {
```

Comment:
lets just delete all of this commented out code -- if we ever need to go back and reference it we can always look at git history